prow/cmd/phaino/main.go

```diff
@@ -57,16 +61,19 @@ func gatherOptions() options {
 	fs.DurationVar(&o.timeout, "timeout", time.Hour, "Maximum duration for each job (0 for unlimited)")
 	fs.DurationVar(&o.totalTimeout, "total-timeout", 0, "Maximum duration for all jobs (0 for unlimited)")
 	fs.DurationVar(&o.grace, "grace", 10*time.Second, "Terminate timed out jobs after this grace period (1s minimum)")
-	fs.StringToStringVar(&o.repoPath, "repo", map[string]string{}, "Local path of repos, can be passed in repeatedly")
+	fs.StringToStringVar(&o.repoPathMap, "repo", map[string]string{}, "Local path of repos, can be passed in repeatedly")
+	fs.BoolVar(&o.mountJobVolumes, "mount-job-volumes", false, "Mount volumes defined in the job spec")
```

Comment:
Should this default to true? Or in another word, this var should be `skip-mount-job-volumes` so that it can be default to be false

Reply:
Other flags are all default to false, I think it's better to be consistent.

I just had a second thought on this - since sometimes users might want to only mount some of the volumes but not all, would it be better to define the flag as a `StringSliceVar` and users can choose which volumes to mount?

Reply:
yeah I would prefer the idea of skipping certain volumes, and by default no one is skipped

Reply:
Added the below 4 flags as we discussed offline:
- `--skip-volume-mounts`
- `--extra-volume-mounts`
- `--skip-envs`
- `--extra-envs`